### DISCUSSION TITLE
Welcome to Gameplay Ingredients Discussions!

### DISCUSSION THREAD

**Discussion by peeweek** [peeweek/net.peeweek.gameplay-ingredients#25] [General]
## 👋 Welcome to the discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Provide constructive feedback about Gameplay Ingredients
  * Ask questions you’re wondering about.
  * Share ideas about UX, workflows and possible new tools.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, you can comment below with an introduction of yourself and tell us about what you do with this community.

## Get engaged with Gameplay Ingredients jolly co-operation

This project is, (always have been in fact) a community project. Some (few) people have already contributed to it. But I humbly ask you, if you make something useful with it and/or want to share some of the improvements. Do not hesitate by making pull requests. 

Right now I am the only maintainer, doing it part-time so I am not always the most active fellow. I could use some help, if you want, join in!

If you want to contribute, I also invite you to read some fundamentals about Release management fundamentals post in discussions.

**Comment by facybenbook** [comment]
every time i run my game i get the message accessing data don't turn off your game , how do i disable that message thank you in advance

**Reply by peeweek** [reply]
Hello, this message is displayed by the Game Save Manager. If you have deployed starter content, you can find it under Assets/Resources. (If you haven't, you can access the setup wizard using Window/Gameplay Ingredients/Setup Wizard (or Welcome Screen on older versions))

If you want to remove this message that appears at the beginning of the game, during the save game loading, you can remove entries of the OnLoad: doing so will prevent the game from displaying the message while loading.

Hope that helps! :)

![image](https://user-images.githubusercontent.com/4037271/120244334-3d8e7400-c26a-11eb-86d3-024d9ed74a1d.png)
![image](https://user-images.githubusercontent.com/4037271/120244386-539c3480-c26a-11eb-89cd-3fe09c60e5cf.png)

(Also, for readability of the conversations, please post a new thread instead of replying in this general message)
